Fixed a bug to process pdf files and added feature to exclude multiple tags.

harun27 opened the pull request.

harun27 committed:
Update README.md

harun27 committed:
fixed a bug for reading pdf text

harun27 committed:
Added possibility to exclude multiple tags

harun27 committed:
Delete rest.py

harun27 committed:
Delete ocr_joplin_notes.py

harun27 committed:
Delete cli.py

harun27 committed:
Delete file_ocr.py

harun27 committed:
Delete joplin_api.py

harun27 committed:
Added possibility to exclude multiple tags

harun27 committed:
Update README.md

harun27 committed:
Update joplin_api.py

plamola commented:
Thanks @harun27 for the pull request.
I'll try to have a proper look at it this weekend (currently moving house)

harun27 committed:
Fixed a bug for reading weird pdfs

Some pdf files have weird characteristics and throw out a PermissionError. I fixed it by tagging those notes with `ojn_ocr_failed`

plamola commented:
Since I refactored the code quite a bit, this pull request had many conflicts. I've manually added the proposed changes to the code base instead.
Closing this pull request.
